content/docs/api-reference/params_show.md

```diff
@@ -0,0 +1,210 @@
+# dvc.api.params_show()
+
+Load <abbr>parameters</abbr> (name and values) tracked in a <abbr>DVC
+project</abbr>.
+
+```py
+def params_show(
+    *targets: str,  # Optional
+    repo: Optional[str] = None,
+    stages: Optional[Union[str, Iterable[str]]] = None,
```

Comment:
This should be the real function signature. Did it change?

Reply:
This is the signature [from what I can tell](https://github.com/iterative/dvc/pull/7613/files#diff-54fd2c4af8ff96a0ddfb2a22c1c56a659c41ae7eb263a6fb943b6d246cb34025R9-R15). I don't think we should alter it for docs (other than that one commend I added).

Reply:
That said if you plan to change the signature in the core repo then sure. Lmk

Reply:
Note that those are all keyword arguments so the order doesn't really affect the behavior. We could also update the order in core, if it makes more sense

Reply:
I don't mind updating in core, but to me it's more important to update here to start with more common use cases.

Reply:
OK. I moved `stages` one position up. Why not all the way up then? Since deem the stage-filtering example most important... Which goes back to https://github.com/iterative/dvc/pull/7613#pullrequestreview-1009761660

> I don't mind updating in core

I think that's ideal because at some point this may get copy-pasted again from there (on some update) and our custom docs order is gone.